cft_sidecar.yaml

```diff
@@ -708,28 +708,6 @@ Resources:
                 docker_setup
                 docker_compose_install
                 update_nginx_resolver
-                echo "Checking for current certs"
-                # Check if a .env file exists already
-                if ! test -f /home/ec2-user/.env; then
-                  GENERATE_TLS="yes"
-                else
-                  # Test to see if the env file contains a Certificate
-                  TEST_FOR_TLS_KEY=`grep '^CYRAL_CERTIFICATE_MANAGER_TLS_KEY=[A-Za-z0-9\/\+]$'  /home/ec2-user/.env`
-                  if [ -z "$TEST_FOR_TLS_KEY" ]
-                  then
-                    GENERATE_TLS="yes"
-                  else
-                    load_certs
-                  fi
-                fi
-
-                if [ $GENERATE_TLS ]
-                then
-                    echo "No certs were found so we'll need to generate them"
-                    generate_certificates "${sidecar_endpoint}" "${sidecar_tls_certificate_secret_arn}" "${sidecar_ca_certificate_secret_arn}"
```

Comment:
from the function file/section remove the `generate_certificates`, `store_cert_data_in_secret` and `put_secret_value` functions

Reply:
Addr [74b0a87](https://github.com/cyral-quickstart/quickstart-sidecar-cloudformation-ec2/pull/8/commits/74b0a8771a80002be20a733e545e5966b669f980)